packages/integration-tests/Collections.js

```diff
@@ -85,6 +85,9 @@ const buildRandomizedCollection = (overrides = {}) => ({
   granuleIdExtraction: '^([^.]+)\..+$',
   sampleFileName: 'asdf.jpg',
   url_path: randomId('url-path-'),
+  meta: {
+    excludeFileTypes: ['.cmr.json', '.cmr.xml', '.h5', '.hdf', '.jpg', '.md5', '.met', '.nc', '.txt'],
```

Comment:
Why was this added?

Reply:
There is no API to disable ORCA for a collection, so add meta.excludeFileTypes to collections for integration tests.
Some tests use the collection returned from buildRandomizedCollection/createCollection  for IngestAndPublishGranule workflow, and IngestAndPublishGranule workflow has CopyToGlacier step.